src/scripts/verify.js

```diff
@@ -0,0 +1,7 @@
+import ControllerFor from '../models/network/ControllerFor'
+
+export default function verify(contractAlias, { network = 'mainnet', txParams = {}, networkFile = undefined, optimizer = false, optimizerRuns = 200, remote = 'etherchain' }) {
```

Comment:
Down the road, we may want to add a flag to verify all contracts (such as a `--all` option, like in the `add` command). Let's make sure that we have an issue that capture this, along with any future improvements we may want to work on.